python/ray/air/predictor.py

```diff
@@ -32,18 +70,57 @@ def from_checkpoint(cls, checkpoint: Checkpoint, **kwargs) -> "Predictor":
         """
         raise NotImplementedError
 
+    @DeveloperAPI
+    @abc.abstractmethod
```

Comment:
Can you move dev APIs to be lower in the file than public APIs, to optimize readability?